Workspaces/Base/Workspace.cs

```diff
@@ -82,12 +84,7 @@ public Bounds outerBounds
 		/// <summary>
 		/// If true, prevent the resizing of a workspace via the front and back resize handles
 		/// </summary>
-		public bool preventFrontBackResize { set { m_WorkspaceUI.preventFrontBackResize = value; } }
-
-		/// <summary>
-		/// If true, prevent the resizing of a workspace via the left and right resize handles
-		/// </summary>
-		public bool preventLeftRightResize { set { m_WorkspaceUI.preventLeftRightResize = value; } }
+		public bool preventResize { set { m_WorkspaceUI.preventResize = value; } }
```

Comment:
Good catch. I also forgot to bring this one up with you. I found that left/right and front/back were always used together, and it simplifies our logic to only use one state. Is there a good reason to control these separately at this point?